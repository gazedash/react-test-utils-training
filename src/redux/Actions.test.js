import { addTodo } from "./Actions";

const payload = {
  error: false,
  id: 0,
  title: "First todo"
};

it("returns todo object", () => {
  expect(addTodo(payload)).toEqual({ payload, type: "ADD_TODO" });
});

it("matches the snapshot", () => {
  expect(addTodo(payload)).toMatchSnapshot();
});