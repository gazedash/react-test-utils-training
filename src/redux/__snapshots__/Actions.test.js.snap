// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
Object {
  "payload": Object {
    "error": false,
    "id": 0,
    "title": "First todo",
  },
  "type": "ADD_TODO",
}
`;
